end2end-test-examples/echo-client/README.md

```diff
@@ -39,6 +39,8 @@ Per sec Payload = 0.07 MB (exact amount of KB = 10000)
 
 `--resSize`: Set response payload size in KB.
 
+`--resType`: Set the response type. If 0, the client sends a `EchoWithResponseSize` rpc. If [1,2,3], the client sends a `BatchEcho` and sets the `response_type` with this value in `BatchEchoRequest`. 
```

Comment:
What do the numbers represent?

Reply:
It's required by Uber team:
 >Type 1 response: A list containing [tps.payload, wp1.payload, wp2.payload]
 Type 2 response: A list containing [tp.payload,  tps.payload, offer.payload, wp1.payload, wp2.payload]
Type 3 response: A list containing [fo.payload, tj.payload, offer.payload,  tps.payload, wp1.payload, wp2.payload]